### PR TITLE
doc: Replace init with migrate, add NOC config info

### LINKED ISSUE #932: Minor changes in Installing The Things Stack documentation
#### Summary
We have identified a few improvements in [Installing The Things Stack documentation](https://www.thethingsindustries.com/docs/getting-started/installation/).
1. Update the Identity Server database initialization command.
2. Add detailed configuration of NOC component.
#### Why do we need this ?
...
#### What is already there? What do you see now?
[Running The Things Stack](https://www.thethingsindustries.com/docs/getting-started/installation/running-the-stack/)
[Configuring The Things Stack](https://www.thethingsindustries.com/docs/getting-started/installation/configuration/)
#### What is missing? What do you want to see?
...
#### How do you propose to document this?
1. Update Identity Server database initialization command.

While starting The Things Stack for the first time, a user needs to initialize the Identity Server database. In the documentation, we see the command `docker-compose run --rm stack is-db init`, which results in the below error.
```
init command deprecated, use migrate instead
ERROR: 255
```
Please update the command below for initializing the Identity Server database.
```
docker-compose run --rm stack is-db migrate
```
2. Add detailed configuration of NOC component.

To provide a better context on the NOC component configuration, please add the configuration details in the [Understanding The Things Stack Configuration](https://www.thethingsindustries.com/docs/getting-started/installation/configuration/#understanding-the-things-stack-configuration) section.
#### Can you do this yourself and submit a Pull Request?
No, @nejraselimovic

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Closes #932 

#### Notes for Reviewers
@johanstokking there's no much documentation available for NOC as far as I can see, so could you add some useful info for this PR if you think it's necessary? I generally think [this doc](https://www.thethingsindustries.com/docs/getting-started/installation/configuration/#understanding-the-things-stack-configuration) could be better, and I see some stuff missing (for example device claiming server config), so I'll make a separate issue for that.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by johanstokking** [comment]
Thanks. Yeah this probably needs some love. I hope to get back on this tomorrow.

**Comment by johanstokking** [comment]
I pushed a commit here with NOC config and database initialization steps.

File: doc/content/reference/configuration/network-operations-center.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Network Operations Center Options"
+description: ""
+distributions: ["Enterprise"]
+---
+
+## Network Operations Center Mount
+
+Network Operations Center can be served under any arbitrary path on your server
```
Comment (adriansmares) [inline doc/content/reference/configuration/network-operations-center.md:9]: Nit: colon at the end.

**Review by adriansmares — APPROVED** [review]

**Review by johanstokking — APPROVED** [review]

**Comment by nejraselimovic** [comment]
Awesome, thanks @johanstokking!

**Pull request merged**